### PR TITLE
Newsletters: add 168 (2021-09-29)

### PR DESCRIPTION
**Pull request by harding**
- [x] *Monday* Stack Exchange section @bitschmidty 
- [x] *Monday* `Bitcoin Core #12677` RPC: Add ancestor{count,size,fees} to listunspent output @jnewbery 
- [x] *Monday* `Eclair #1942` Add success probabilities in path finding @dongcarl
- [x] *Tuesday* Update releases/RCs and add topic links @harding

### CODE REVIEW

**Comment by jnewbery** [comment]
Reviewed everything and pushed a commit for Bitcoin Core 12677. All looks good so far!

File: _posts/en/newsletters/2021-09-29-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Why TXID has 256 bits when Bitcoin's address security is 160-bit?]({{bse}}109652)
```
Comment (Xekyo) [inline _posts/en/newsletters/2021-09-29-newsletter.md:74]:
```suggestion
- [Why does the TXID have 256 bits when Bitcoin's address security is 160-bit anyway?]({{bse}}109652)
```
I edited the stack exchange post for grammar

**Review by Xekyo — COMMENTED** [review]
Thanks for reminding people about BIP322. I found a few typos. I also edited the question with wonky grammar on Stack Exchange and updated the title here.

**Comment by bitschmidty** [comment]
@Xekyo Thanks for the feedback, pushed a commit to resolve.

File: _includes/specials/taproot/en/14-signmessage.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+widely accepted way to create signed text messages for [bech32 or
+bech32m][topic bech32] addresses.  Arguably, that means we can now
+assume that widespread message signing support must not be very
+important to users or developers, otherwise more work would've been
+dedicated to it.  But it still feels like Bitcoin wallet software has
+regressed since the days when everyone used legacy addresses and could
+easily trade signed messages.
+
+The [generic signmessage][topic generic signmessage] solution we wrote
+about two years in our [bech32 spending support][bech32ss signmessage]
```
Comment (bitschmidty) [inline _includes/specials/taproot/en/14-signmessage.md:11]: s/two years in/two years ago in

File: _posts/en/newsletters/2021-09-29-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+  routing success rates.
+
+- [LND #5101][] adds a *middleware interceptor* which receives each RPC
+  request on its way to the server and can make modifications.  This
+  allows implementing logic outside of LND that can track or affect a
+  large variety of user and automated actions.  For security, only RPCs
+  that explicitly use authentication tokens (macaroons) that opt-in to
+  interception can be intercepted.
+
+- [BTCPay Server #2842][] adds the ability to disable (and later
```
Comment (bitschmidty) [inline _posts/en/newsletters/2021-09-29-newsletter.md:149]: I think we could drop this commit from the newsletter. I do not feel strongly about it, however.

Comment (harding) [reply _posts/en/newsletters/2021-09-29-newsletter.md:149]: Yeah, I was on the edge about this, thanks for weighing in!

**Comment by harding** [comment]
@bitschmidty I added a FIXME for you pre-merge Wednesday, if you don't mind checking to see if the link for Bitcoin Core 0.20.2 has been uploaded or not.  If it has, please remove the extraneous sentence.  Thanks!

Made edits for all feedback (hopefully), updated releases, added topic links, and reviewed @jnewbery, @dongcarl, and @bitschmidty contributions (great, as always!).  Thanks everyone!

**Comment by jnewbery** [comment]
re-reviewed everything and all looks good to me. Thanks everyone!

One final FIXME to address before merge:

https://github.com/bitcoinops/bitcoinops.github.io/pull/654/files#diff-88183ff1f3ec5dcd1e2980fadd796607850858e8c9aa39473a2b5bd4469dcb30R116

**Comment by bitschmidty** [comment]
Squashed and merged! Thanks @harding @dongcarl @jnewbery @Xekyo !

**Pull request merged**